From 85af2287f20a21fce2373a7a866e813ba215b899 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:05:07 -0400
Subject: [redhat] kabi: Adding symbol pv_mmu_ops (arch/x86/kernel/paravirt.c)

Message-id: <56d4f4f64050e8e05eb4a0095631ee289633b3d8.1412771859.git.poros@redhat.com>
Patchwork-id: 97200
O-Subject: [RHEL7.1 PATCH 47/67] kabi: Adding symbol pv_mmu_ops (arch/x86/kernel/paravirt.c)
Bugzilla: 1148030 1148038
RH-Acked-by: Andrew Jones <drjones@redhat.com>

Bugzilla: 1148030,1148038
    https://bugzilla.redhat.com/show_bug.cgi?id=1148030
    https://bugzilla.redhat.com/show_bug.cgi?id=1148038

    Description:
    ============
    These symbols are required so that SGI modules play
    nice with paravirt in the kernel; yes we are trying
    to make guest OS more Numa aware.
    Symbol used in gru.ko, numatools.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pv_mmu_ops

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pv_mmu_ops b/redhat/kabi/kabi-module/kabi_x86_64/pv_mmu_ops
new file mode 100644
index 0000000..d1eaf84
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pv_mmu_ops
@@ -0,0 +1,2 @@
+#1-
+0x72392fe6 pv_mmu_ops vmlinux EXPORT_SYMBOL
-- 
1.7.1